### PR TITLE
Fix #16845 Add links to URL system

### LINKED ISSUE #16845: Links not passing through url.php
### Describe the bug

I've noticed that a few links don't pass through url.php

### To Reproduce

1. Go to https://demo.phpmyadmin.net/master-config/index.php?route=/
2. See the links for `RELEASE_5_1_0-841-ge0c1dea460` and `master`, found at the bottom of the page
3. See the link for `5.1.0` found on the right side on version information

### Expected behavior

The links should pass through url.php.

### Client configuration

- Browser: Firefox
- Operating system: Windows

### Additional context
https://github.com/phpmyadmin/phpmyadmin/blob/e0c1dea460c631358ccdacddc0c22d3ed8e0cf8c/libraries/classes/Command/WriteGitRevisionCommand.php#L51
https://github.com/phpmyadmin/phpmyadmin/blob/e0c1dea460c631358ccdacddc0c22d3ed8e0cf8c/libraries/classes/Command/WriteGitRevisionCommand.php#L58
https://github.com/phpmyadmin/phpmyadmin/blob/e0c1dea460c631358ccdacddc0c22d3ed8e0cf8c/js/src/home.js#L44

### PR DESCRIPTION
**Pull request by liviuconcioiu**
Signed-off-by: Liviu-Mihail Concioiu <liviu.concioiu@gmail.com>

### Description

I have changed the links that didn't go through the URL system.

Fixes #16845 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

File: libraries/classes/Command/WriteGitRevisionCommand.php [hunk 1]
```diff
@@ -48,14 +49,14 @@ protected function configure(): void
...
-            'https://github.com/phpmyadmin/phpmyadmin/commit/%s'
+            Core::linkURL('https://github.com/phpmyadmin/phpmyadmin/commit/%s')
         );
         $this->addOption(
             'remote-branch-url',
             null,
             InputOption::VALUE_OPTIONAL,
             'The remote URL to a branch',
-            'https://github.com/phpmyadmin/phpmyadmin/tree/%s'
+            Core::linkURL('https://github.com/phpmyadmin/phpmyadmin/tree/%s')
```
Comment (williamdes) [inline libraries/classes/Command/WriteGitRevisionCommand.php:59]: I would not do that here but rather in the `Footer::getDisplay` method

**Comment by williamdes** [comment]
> This commit has an invalid Signed-Off-By line, please use your full name for legal compliance reasons.
> Please check [Developer's Certificate of Origin](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO)
> Here is a valid example: `Signed-off-by: Jane Smith <jane.smith@example.org>`

Please use you your real name like in previous PRs :)

File: libraries/classes/Command/WriteGitRevisionCommand.php (hunk 1 above)
Comment (liviuconcioiu) [reply libraries/classes/Command/WriteGitRevisionCommand.php]: Done.

**Comment by liviuconcioiu** [comment]
> Please use you your real name like in previous PRs :)

Done! ;)

File: libraries/classes/Footer.php [hunk 2]
```diff
@@ -320,7 +320,7 @@ public function getDisplay(): string
             if (! $this->isAjax && ! $this->isMinimal) {
                 if (Core::getenv('SCRIPT_NAME')) {
                     $url = $this->getSelfUrl();
-                    $selfLink = $this->getSelfLink($url);
+                    $selfLink = $this->getSelfLink(Core::linkURL($url));
```
Comment (williamdes) [inline libraries/classes/Footer.php:323]: I think you did a mistake, because this change is before `$gitRevisionInfo = $this->getGitRevisionInfo();` :/

Comment (liviuconcioiu) [reply libraries/classes/Footer.php]: The new fix is working.

**Review by williamdes — APPROVED** [review]
Nice work !

**Pull request merged**